e2e/next/src/next-styles.test.ts

```diff
@@ -5,7 +5,7 @@ describe('Next.js apps', () => {
   let originalEnv: string;
 
   beforeAll(() => {
-    newProject();
+    newProject({ packageManager: 'npm' });
```

Comment:
Why only `npm`?

Reply:
Still trying to debug why this e2e test keeps failing in CI